db/migrate/20220701182837_add_custom_theme_id_to_adventures.rb

```diff
@@ -0,0 +1,8 @@
+class AddCustomThemeIdToAdventures < ActiveRecord::Migration[5.2]
+  def change
+    add_column :adventures, :custom_theme_id, :integer
```

Comment:
This is set to optional in adventure.rb, so this should probably have a null:true, as I believe the default is null:false, so it is not actually optional.